test/http.js

```diff
@@ -546,6 +546,29 @@ describe('HTTP APIs', function () {
         .expect(200, done)
     })
 
+    it('should create a container with a name hex decoded from the slug', (done) => {
+      let containerName = 'Film%4011'
+      let expectedDirName = '/post-tests/Film@11/'
+      server.post('/post-tests/')
+        .set('slug', containerName)
+        .set('content-type', 'text/turtle')
+        .set('link', '<http://www.w3.org/ns/ldp#BasicContainer>; rel="type"')
+        .expect(201)
+        .end((err, res) => {
+          if (err) return done(err)
+          try {
+            assert.equal(res.headers.location, expectedDirName,
+              'Uri container names should be encoded')
+            let createdDir = fs.statSync(path.join(__dirname, 'resources', expectedDirName))
+            assert(createdDir.isDirectory(), 'Container should have been created')
+          } catch (err) {
+            return done(err)
+          }
+          done()
+        })
+    })
+
+    /* No, URLs are NOT ex-encoded to make filenames -- the other way around.
```

Comment:
This test should be changed to test for a 400 error response, per our conversation.